src/client.ts

```diff
@@ -0,0 +1,24 @@
+import { Connection } from "./Connection";
+import { AuctionHouseClient } from "./auctionHouse";
+
+export class MetaplexClient {
+    private auctionHouseClient: AuctionHouseClient;
+    // private metadataClient: MetadataClient; EXAMPLE OF OTHER FEATURES
+
+    constructor(private connection: Connection) { }
+
+    get AuctionHouse() {
```

Comment:
Why does this have to live on the client instead of just being returned?
i.e. if I'm done with it the GC cannot collect it now because it's still referenced.
Incurring that memory overhead only makes sense if creating it is expensive (which I don't think it is given you're just calling a constructor).